src/qt/qt_mediamenu.cpp

```diff
@@ -662,6 +676,30 @@ void MediaMenu::moUpdateMenu(int i) {
     menu->setTitle(QString::asprintf(tr("MO %i (%ls): %ls").toUtf8().constData(), i + 1, busName.toStdU16String().data(), name.isEmpty() ? tr("(empty)").toStdU16String().data() : name.toStdU16String().data()));
 }
 
+void MediaMenu::nicUpdateMenu(int i) {
+    if (!netMenus.contains(i))
+        return;
+
+    QString netType = tr("None");
+    switch (net_cards_conf[i].net_type) {
+        case NET_TYPE_SLIRP:
+            netType = "SLiRP";
+            break;
+        case NET_TYPE_PCAP:
+            netType = "PCAP";
+            break;
+    }
+
+    QString devName = DeviceConfig::DeviceName(network_card_getdevice(net_cards_conf[i].device_num), network_card_get_internal_name(net_cards_conf[i].device_num), 1);
+
+    auto *menu = netMenus[i];
+    auto childs = menu->children();
+    auto *connectedAction = dynamic_cast<QAction*>(childs[netDisconnPos]);
+    connectedAction->setChecked(!network_is_connected(i));
```

Comment:
```suggestion
    connectedAction->setChecked(network_is_connected(i));
```
This is just to support the above suggestion of defaulting to "Connected" and enabled in the menu.